manifests/generated/rbac.authorization.k8s.yaml.in

```diff
@@ -187,6 +187,7 @@ rules:
   - virtualmachineinstancemigrations
   verbs:
   - get
+  - create
```

Comment:
this file is generated, you need to make this change in /pkg/virt-operator/creation/rbac/apiserver.go